### PR TITLE
Fixed image copy issue #3932

### PR DESCRIPTION
**Pull request by deepaksinghgusain227**
BUGS: Fixed

Fixed issued #3932 

Previously
When Admin want to copy the product then the image section not copy to new directory instead it link to that product id
which the product you copied

Now
when admin wants to copy a product then the image section make a new copy and image get uploaded to that new directory.
No link to previous product id

### CODE REVIEW

File: packages/Webkul/Product/src/Repositories/ProductRepository.php [hunk 1]
```diff
@@ -750,4 +751,18 @@ private function persistRelations($originalProduct, $copiedProduct): void
             ]);
         }
     }
+
+    public function copyImage($image,$copiedProduct) {
+
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Repositories/ProductRepository.php:756]: Remove one line from here.

**Comment by vaishaliwebkul** [comment]
@deepaksinghgusain227 
while copying a product, getting an issue
Screenshot_120

**Comment by vaishaliwebkul** [comment]
@deepaksinghgusain227 
remove .env.example file from changed file.

**Comment by vaishaliwebkul** [comment]
@deepaksinghgusain227 
getting another exception again https://prnt.sc/va0uze while copying product

**Comment by vaishaliwebkul** [comment]
closing this as already resolved with this PR https://github.com/bagisto/bagisto/pull/4300

**Pull request closed without merging**